src/CMakeLists.txt

```diff
@@ -497,15 +524,10 @@ elseif(${COMPILER_ARCHITECTURE} STREQUAL "x86_64")
     set(_sse_flag -msse4.1)
   endif()
 
-  set_source_files_properties(../auxil/highwayhash/highwayhash/hh_avx2.cc PROPERTIES COMPILE_FLAGS
+  set_source_files_properties(${hhash_dir}/hh_avx2.cc PROPERTIES COMPILE_FLAGS
                               ${_avx_flag})
-  set_source_files_properties(../auxil/highwayhash/highwayhash/hh_sse41.cc PROPERTIES COMPILE_FLAGS
+  set_source_files_properties(${hhash_dir}/hh_sse41.cc PROPERTIES COMPILE_FLAGS
                               ${_sse_flag})
-
-  list(APPEND HH_SRCS
-      ../auxil/highwayhash/highwayhash/hh_avx2.cc
-      ../auxil/highwayhash/highwayhash/hh_sse41.cc
-  )
 endif ()
```

Comment:
Since `highwayhash` is a fork of a 3rd-party repository that doesn't use CMake, I think it's fine doing it that way. That being said, maybe they would be open to a contribution for CMake in their repository as a better mid-term solution?